aea/cli/check_packages.py

```diff
@@ -54,6 +55,7 @@
     DEFAULT_CONNECTION_CONFIG_FILE,
     DEFAULT_CONTRACT_CONFIG_FILE,
     DEFAULT_PROTOCOL_CONFIG_FILE,
+    DEFAULT_CUSTOM_COMPONENT_CONFIG_FILE,
```

Comment:
Custom component config file reference was missing from the check-packages command